clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -706,6 +706,8 @@ private void checkDescribeConfigsResponseVersions() {
     private void checkErrorResponse(AbstractRequest req, Throwable e, boolean checkEqualityAndHashCode) {
         AbstractResponse response = req.getErrorResponse(e);
         checkResponse(response, req.version(), checkEqualityAndHashCode);
+        Map<Errors, Integer> errorCounts = response.errorCounts();
+        assertTrue(errorCounts.containsKey(Errors.forException(e)), "API Key " + req.apiKey().name + "V" + req.version() + " failed errorCounts test");
```

Comment:
Is it possible to make this assertion stronger? Would we expect any other errors in the response?

Reply:
@hachikuji Thanks for the review! I pushed an update